admin2.php

```diff
@@ -120,11 +120,11 @@
 						<!-- Form for input fields -->
 						<form method = "post" id= "01_login">
 
-							<center><input style = "width: 20vw; height: 1.9vw; margin: 2vw 0vw 0vw 0vw; font-family: 'Russo One', sans-serif; color: #000;" type = "text" name = "existing_username" placeholder="Existing Username..."/></center>
+							<center><input style = "width: 20vw; height: 1.9vw; margin: 2vw 0vw 0vw 0vw; font-family: 'Roboto', sans-serif; color: #fff;" type = "text" name = "existing_username" placeholder="Existing Username..."/></center>

```

Comment:
Majority consists of applying "Roboto" as the heading font and changing the font colour to white from here and below.